src/response.rs

```diff
@@ -155,14 +233,14 @@ impl Response {
     /// let string: String = res.body_string().await?;
     /// # Ok(()) }
     /// ```
-    pub async fn body_string(&mut self) -> Result<String, Error> {
+    pub async fn body_string(&mut self) -> http_types::Result<String> {
```

Comment:
Is `surf::Result` not available? That should probably be a re-export of `http_types::Result`.

Reply:
Tide's is not a re-export, I'm going to change it to be like Tide: https://github.com/http-rs/tide/blob/e994b2ebbaa5a1ebe6849bff720adff89dae1fb5/src/lib.rs#L286-L287